src/value/owned.rs

```diff
@@ -506,6 +506,74 @@ mod value_tests {
         assert_eq!(actual, expected);
     }
 
+    /// Each case is a set of tokens that are the same, and a set of tokens that are not ever equal to the first.
+    /// This should test symmetry/transitivity/commutativity
+    #[rstest(
+        // SymbolTokens with same text are equivalent
+        case::sym_text(
+            vec![
+                text_token("foo"),
+                text_token("foo").with_local_sid(10),
+                text_token("foo").with_local_sid(10).with_source("greetings", 2)
+            ],
+            vec![
+                text_token("foo"),
```

Comment:
This doesn't look right, how come `text_token("foo")` in the second vector is not equivalent to the first set of `"foo"` tokens?